.release-please-manifest.json

```diff
@@ -8,5 +8,6 @@
     "providers/go-feature-flag": "0.1.29",
     "providers/flagsmith": "0.1.4",
     "providers/launchdarkly": "0.1.3",
-    "tests/flagd": "1.3.1"
+    "tests/flagd": "1.3.1",
+    "tests/unleash": "0.0.1-alpha"
```

Comment:
@liran2000 I guess this was a copy paste error, and you mean to do `providers/unleash`. I will force a quick fix for this, right now this is causing a 1.0 release which I don't think you want.

Sorry I missed this! :sweat_smile:

Reply:
Fixed and release looks good now: https://github.com/open-feature/go-sdk-contrib/pull/346